client/src/pages/FilterPage.js

```diff
@@ -1,20 +1,210 @@
 import styled from "styled-components/macro";
+import PropTypes from "prop-types";
 import { CloseButton } from "../components/IconButton.js";
+import { useState } from "react";
 
-const PageWrapper = styled.div`
-  width: 100%;
-  height: 100vh;
+const FilterPageWrapper = styled.div`
```

Comment:
maybe you could add    ``` overflow-y: scroll``` to the FilterPageWrapper?

Reply:
Thank you, great hint! It now works with `overflow: hidden` 🙂👍